model/client4.go

```diff
@@ -4508,6 +4508,31 @@ func (c *Client4) DisablePlugin(id string) (bool, *Response) {
 	return CheckStatusOK(r), BuildResponse(r)
 }
 
+// GetMarketplacePlugins will return a list of plugins that an admin can install.
+// WARNING: PLUGINS ARE STILL EXPERIMENTAL. THIS FUNCTION IS SUBJECT TO CHANGE.
+func (c *Client4) GetMarketplacePlugins(request *MarketplacePluginFilter) ([]*MarketplacePlugin, *Response) {
```

Comment:
When we call this function in `api4/plugin.go` we pass a variable `filter` as argument, as I would expect.  
Here the formal parameter of the function is named `request`.  
It is a minor thing but this naming choice might result to be confusing.

Reply:
cc @ali-farooq0

Reply:
ah yes, it was originally a `request` type of struct, but when i switched it, forgot to the update the name. Will fix. Good catch!